lsp-completion.el

```diff
@@ -465,7 +465,7 @@ The MARKERS and PREFIX value will be attached to each candidate."
           ;; metadata
           ((equal action 'metadata)
            `(metadata (category . lsp-capf)
-             (display-sort-function . identity)))
+                      (display-sort-function . identity)))
```

Comment:
This indentation is intended as a plist. You should not change this or add `( metadata` so it will be aligned

Reply:
This is not `plist`.
Maybe you want this?
```diff
          (cond
           ;; metadata
           ((equal action 'metadata)
-           `(metadata (category . lsp-capf)
-                      (display-sort-function . identity)))
+           `(metadata
+             (category . lsp-capf)
+             (display-sort-function . identity)))
```

Reply:
Ah, nvm, the change on this looks good with 
``` elisp
           `(metadata (category . lsp-capf)
                      (display-sort-function . identity)))
```

Reply:
? I cannot understand `nvm` but, finally, you approve this?

Reply:
`nvm` is stand for `nevermind`.

Reply:
Thanks, i memorize it.